docs/1-Using-Fleet/2-fleetctl-CLI.md

```diff
@@ -348,6 +349,15 @@ spec:
       removed: false
 ```
 
+#### Moving queries and packs from one Fleet environement to another
+
+When managing multiple Fleet environments, you may want to move queries and/or packs from one environment (exporter) to the other (importer).
+
+1. With your exporter Fleet environment running, run `fleetctl get queries --yaml > queries.yml`. This will write a list of all queries in yaml syntax to a file named queries.yml.
```

Comment:
Somewhere in here it would be good to mention that you can use "contexts" feature of fleetctl to support the multiple environments. Users can do something like `fleetctl get queries --yaml --context exporter > queries.yaml && fleetctl apply -f queries.yml --context importer`.

Reply:
Aha! That's significantly simpler. Going to update the documentation in this PR and the blog post.

Reply:
@zwass I adjusted the instructions to include the contexts feature. Also tested this feature on my local Fleet instance -> Fleet preview instance. Very sweet.